### ISSUE TITLE
Parameter for Custom Details

### ISSUE THREAD

**Issue by denitzavl** [alexanderlz/jenkins-pagerduty-plugin#50]
Due to a recent change in the plugin we are not able to send custom details to PagerDuty. Some parameters were changed and we are unaware what new parameter we need to set to have additional details to be shown. 

Below you can see how was our configuration prior and after the recent change.
Before this we were using **incDetails** to provide more information to PagerDuty and directly into the incident.

Here is how it looks on Jenkins side:

Before:
pagerduty(resolve: false, **serviceKey**: "***********", **incDescription**: "Service not available", **incDetails**: "Triggering Jenkins job: https://test/ \nUrl which was checked: https://service-endpoint/")

After/ Current:
pagerduty(resolve: false, **routingKey**: "***********", **incidentSummary**: "Service not available", **incidentSource**: "Triggering Jenkins job: https://test/ \nUrl which was checked: https://service-endpoint/")

Here is how the message payload is seen in PagerDuty:

**Before:**
{
  "client": "JenkinsPagerDutyClient",
  "description": "Service not available",
  "event_type": "trigger",
  "incident_key": "**********************",
  "service_key": "***********",
  "details": "Triggering Jenkins job: https://test/ 
Url which was checked: https://service-endpoint/"
}

Here is how it looks after the change and as we can see although that from Jenkins we are sending details as “incidentSource” we are not getting the details as previously.

**After/ Current:**
{
  "contexts": [],
  "description": "Service not available",
  "event_type": "trigger",
  "incident_key": "**********************",
  "service_key": "***********"
}

Here is a screenshot of the difference seen in the incidents directly in PagerDuty:

•	On the left side we see that the Custom Details are included - this is with the **Before** configuration from above 
•	On the right side we see the Custom Details missing – this is with the **After/Current** configuration from above, where we had to change the parameters, etc.

![Inkedjj-pd_LI](https://user-images.githubusercontent.com/78234397/106326537-591a4b80-6285-11eb-8646-aac3a125d8e6.jpg)

After checking with PagerDuty support they told us that on their end they are receiving **"custom_details":null** from Jenkins. Which means that the incidentSource is not doing the job that we need. 

All in all - we cannot find what new parameter to use to achieve the previous configuration with inclusive custom details.

Not sure how related is to https://github.com/alexanderlz/jenkins-pagerduty-plugin/issues/45 

Thank you.

**Comment by akrapfl** [comment]
I don't think it is related to #45, as that change did not change the `pagerDuty` step.   I know  that the params were changed recently, but I suppose you already know that part.

**Comment by alexanderlz** [comment]
@denitzavl - thanks for bringing up this issue. I will take a look on it and hopefully will release an update to pluginCenter during this week.

@akrapfl your PRs are planned there too

**Comment by denitzavl** [comment]
@alexanderlz , awesome! Thank you very much.

Let me know if you need me to provide any additional details .

**Comment by denitzavl** [comment]
Hello @alexanderlz , @akrapfl ,

Just checking in on what is the current status of the issue. 

Thank you!
All the best,
-Denitsa

**Comment by denitzavl** [comment]
Hey guys,
Any news from your side?

Thank you!

**Comment by alexanderlz** [comment]
@denitzavl Hi
can you please check if this hpi contains description?

https://github.com/alexanderlz/jenkins-pagerduty-plugin/blob/alexanderlz-patch-1/pagerduty_6_3_alpha.hpi

can you text me on gitter?

**Comment by rudrasish27** [comment]
@alexanderlz akrapfl
I am also facing similar issue. Do we have any update on the custom details parameter?
